src/lib/vocab/random.ts

```diff
@@ -47,15 +47,15 @@ export const randomCommunityParams = (persona_id: number): CreateCommunityParams
 };
 export const randomSpaceParams = (community_id: number): CreateSpaceParams => ({
 	community_id,
-	content: randomContent(),
+	content: JSON.stringify(randomData()),
```

Comment:
```suggestion
	content: randomString(),
```

This is clearer, since `randomData` is different than the JSON content here, which will be migrated to `view` soon.